src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            if (figures.length / 2 > i) {
+                figures[i] = figureSupplier.getRandomFigure();
+                figures[i].draw();
+            } else {
+                figures[i] = new Square(Color.RED.name(), 5);
```

Comment:
This should be done according to task description, not hardcoded:
"For this purpose create a new method in the FigureSupplier class:
public Figure getDefaultFigure() - this method should always return white circle with radius 10."